04/app.js

```diff
@@ -6,4 +6,16 @@ const user = {
         month: '04',
         year: '1985'
     }
-}
\ No newline at end of file
+}
+
+const currentDate = new Date(1985, 4, 14);
+
+if (
+    currentDate.getDate() === Number(user.born.day) &&
+    currentDate.getMonth() === Number(user.born.month) &&
+    currentDate.getFullYear() === Number(user.born.year)
```

Comment:
Czy faktycznie rok musi być ten sam? Wtedy tylko w momencie narodzin będzie to rozwiązanie działać ;)
My chcemy obchodzić urodziny co roku! :D